src/app/pages/home/footer/footer.component.scss

```
@@ -0,0 +1,15 @@
+.bg-dark {

```

If you want the background of the footer to be black, you should use the css property `background-color` instead and set it as `black` with the `!important` flag.
```suggestion
.bg-dark {
  backround-color: black !important;
}
```